src/less/components/input-group.less

```
@@ -12,38 +12,15 @@
     width: 100%;
     min-height: 2rem;
 
-    &:focus-visible {
-        outline: 2px solid @brand-secondary;
-        border-radius: 2px;
-    }
-
-    &:focus-within {
-        outline: 2px solid @brand-secondary;
-
-        .input-group-addon,
-        .form-control {
-            outline: none;
-            border-width: 1px 1px 2px;
-            border-color: transparent;
-            transition: none;
-        }
-
-        .postfix {
-            border-right: 2px solid @black;
-            border-left: transparent;
-            border-top-left-radius: 0;
-        }
-    }
-
     > .btn {
         border-radius: 0;
 
         &:first-child {
-            border-radius: @btn-border-radius-base 0 0 @btn-border-radius-base;
+            border-radius: 8px 0 0 8px;
```

use the css variable?
```suggestion
            border-radius: var(--border-radius, 8px) 0 0 var(--border-radius, 8px);
```